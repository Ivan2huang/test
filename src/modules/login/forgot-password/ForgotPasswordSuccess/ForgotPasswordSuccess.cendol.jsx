/* eslint-disable react/jsx-one-expression-per-line */

import React from 'react';

import ForgotPasswordSuccessContent from '../ForgotPasswordSuccessContent';

const ForgotPasswordSuccess = props => {
  return (
    <>
      <ForgotPasswordSuccessContent showBackBtn={false} {...props} />
    </>
  );
};

export default ForgotPasswordSuccess;
