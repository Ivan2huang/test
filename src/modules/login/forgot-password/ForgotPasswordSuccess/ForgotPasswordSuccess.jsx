/* eslint-disable react/jsx-one-expression-per-line */

import React from 'react';

import ForgotPasswordSuccessContent from '../ForgotPasswordSuccessContent';

const ForgotPasswordSuccess = props => {
  return (
    <>
      <ForgotPasswordSuccessContent {...props} />
    </>
  );
};

export default ForgotPasswordSuccess;
