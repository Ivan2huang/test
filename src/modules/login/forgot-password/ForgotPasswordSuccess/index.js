import DefaultForgotPassword from './ForgotPasswordSuccess';
import CendolForgotPassword from './ForgotPasswordSuccess.cendol';
import BasilForogtPassword from './ForgotPasswordSuccess.basil';
import CoreForgotPassword from './ForgotPasswordSuccess.core';
import CONFIG from '../../../../constants/config';

const containers = {
  cendol: CendolForgotPassword,
  balboa: DefaultForgotPassword,
  basil: BasilForogtPassword,
  core: CoreForgotPassword,
};

const container = containers[CONFIG.themeCode] || DefaultForgotPassword;

export default container;
