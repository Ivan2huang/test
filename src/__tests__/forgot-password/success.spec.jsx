import React from 'react';
import { render } from '@testing-library/react';

import ForgotPasswordSuccess from '../../pages/forgot-password/success';

jest.mock('../../modules/login/forgot-password', () => ({
  ForgotPasswordSuccessContainer: () => (
    <div>Forgot Password Success Component</div>
  ),
}));

jest.mock('../../layouts/withLoginLayoutProvider', () => Component => () => (
  <div>
    <header>Layout Header</header>
    <Component />
  </div>
));

describe('Forgot Password Success Page', () => {
  it('should match snapshot', () => {
    const { container } = render(<ForgotPasswordSuccess />);
    expect(container).toMatchSnapshot();
  });
});
