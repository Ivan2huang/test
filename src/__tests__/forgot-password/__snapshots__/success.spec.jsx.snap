// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forgot Password Success Page should match snapshot 1`] = `
<div>
  <div>
    <header>
      Layout Header
    </header>
    <div>
      Forgot Password Success Component
    </div>
  </div>
</div>
`;
